src/builder.rs

```diff
@@ -69,119 +80,65 @@ pub enum OptionsError {
     PkiError(#[from] webpki::Error),
 }
 
-/// Structure used to store the configuration options for an instance of
-/// [AstarteDeviceSdk][crate::AstarteDeviceSdk].
-#[derive(Clone)]
-pub struct AstarteOptions<S> {
-    pub(crate) realm: String,
-    pub(crate) device_id: String,
-    pub(crate) credentials_secret: String,
-    pub(crate) pairing_url: String,
-    pub(crate) interfaces: Interfaces,
-    pub(crate) store: S,
-    pub(crate) ignore_ssl_errors: bool,
-    pub(crate) keepalive: std::time::Duration,
+pub trait DeviceBuilderBuild<S, C> {
```

Comment:
I would like to choose a better name:
- `AstarteDeviceSdkBuilder`
- `AstarteDeviceSdkBuild`
- `SdkBuild`
- ...

Reply:
As per previous discussion I choose the `DeviceSdkBuild` name.